### ISSUE TITLE
Add path drawing tool

### ISSUE THREAD

**Issue by GMartigny** [GMartigny/pencil.js#5]
Adding shape with a single class instantiation is nice, but sometimes you could want more in-depth drawing.

Could be a declarative set of rules

```js
// Easy to read but blotted
const startingPoint = new Position();
new Shape(startingPoint, [
	Shape.lineTo(position),
	Shape.arcTo(position, center),
	Shape.quadTo(position, controlPoint),
	Shape.bezierTo(position, controlPoints),
], options);
```

Or a set of code like SVG

```js
// Clean but hard to read
const startingPoint = new Position();
new Shape(startingPoint, "M 20 230 Q 40 205, 50 230 T 90230", options);
```

**Comment by GMartigny** [comment]
I'll go with the first solution 'cause I find the second to cryptic. If someone think differently, please tell me.

**Comment by GMartigny** [comment]
Fixed by 7c89491a87c994d39f1e42cb0eb395b1d2b25775
